scss/form/select.scss

```diff
@@ -0,0 +1,86 @@
+.nes-select {
+  // prettier-ignore
+  $dropdown: (
+    (1,1,1,1,1,1,1),
+    (1,1,1,1,1,1,1),
+    (0,1,1,1,1,1,0),
+    (0,1,1,1,1,1,0),
+    (0,0,1,1,1,0,0),
+    (0,0,0,1,0,0,0),
+  );
+
+  position: relative;
+  width: 100%;
+
+  select {
+    width: 100%;
+    padding: 0.5rem 2.5rem 0.5rem 1rem;
+    -webkit-appearance: none;
+    appearance: none;
+    cursor: $cursor-click-url, pointer;
+    border: none;
+    border-radius: 0;
+    box-shadow: 0 4px $color-black, 0 -4px $color-black, 4px 0 $color-black, -4px 0 $color-black;
+
+    &:invalid {
+      color: $color-00;
+    }
+  }
+
+  &::after {
+    $colors: ($base-color, map-get($default-colors, "shadow"));
+
+    @include pixelize($dropdown, $colors);
+
+    position: absolute;
+    top: calc(50% - 11px);
+    right: 36px;
+    width: 2px;
+    height: 2px;
+    font-size: 2px;
+    content: "";
+    transform: scale(1.5);
```

Comment:
`pixelize` will take care of setting width and height of this element for the desired pixel size based on the font size and should not be set explicitly here. Also if you set font-size to 3px you will not need the `transform: scale(1.5)`

Reply:
Oh wow, it does work 🎉 I did spend a while trying to find a sweet spot for values of `width`, `height`, `font-size` and `scale` in order to make it look good.. 😅